### ISSUE TITLE
Getting errors while fitting the model on data

### ISSUE THREAD

**Issue by ajinkya2903** [UKPLab/sentence-transformers#496]
Hey, I am trying to fine tune distil bert nli model on my custom dataset. Even after creating the data in the same format the way model requires it is giving some errors which I am unable to solve. Can you help me in the way so that I could finetune the model properly. I have attached ScreenShot of the error I am getting. PLease do let me know if anything required from my side?
![image](https://user-images.githubusercontent.com/63385842/95945988-4597c180-0e09-11eb-9ad0-0c157d87e043.png)

**Comment by nreimers** [comment]
The error indicates that you uses a train_loss, that requires float labels. But you provide int labels in your training data.

If you change the labels from int to float, it should work.
